docker/init.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+pid_file='/var/run/gunicorn/gunicorn.pid'
+
+function stop_container {
+    service nginx stop
+    gunicorn_pid=$(cat ${pid_file})
+    kill -TERM $(cat ${pid_file})
+}
+
+trap stop_container SIGTERM
```

Comment:
Maybe document why you need to do this?

Reply:
Yeah, I guess this deserves a comment; most people probably just let their container get killed.

Reply:
Added one...